CGPs/cgp-0134.md

```diff
@@ -0,0 +1,108 @@
+---
+cgp: 0134
+title: "Validators rewards - changes in epoch parameters"
+date-created: 2024-05-04
+author: '@Thylacine @Roman @Patrick @Dee>'
```

Comment:
I have withdrawn from this workstream so please remove me from the set of authors.

Reply:
Apologies @rcroessmann. You have been removed